ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/SourceSchemas.java

```diff
@@ -42,7 +42,7 @@
       final Map<SourceName, LogicalSchema> sourceSchemas,
       final KsqlConfig ksqlConfig) {
     this.sourceSchemas = ImmutableMap.copyOf(requireNonNull(sourceSchemas, "sourceSchemas"));
-    this.ksqlConfig = ksqlConfig;
+    this.ksqlConfig = requireNonNull(ksqlConfig);
```

Comment:
Use the two-arg version of `requireNonNull` instead, and same for all the other places (in other classes) you've added the one-arg version as well.